src/lib/notifications/notifications.ts

```diff
@@ -81,15 +81,14 @@ export function registerTokenChangeHandler(
 
 export function init(queryClient: QueryClient) {
   // handle notifications that are received, both in the foreground or background
+  // TODO: is this needed?
```

Comment:
Doesn't seem like it. If someone taps on a push, we handle the invalidation and such then. No need to do anything ahead of time right now. Maybe keep this around just for debug logging?

Reply:
yeah that's what I'm thinking